### PR TITLE
feat: remove members from sponsor dashboard

### PR DESCRIPTION
**Pull request by Sajal0208**
## What does this PR do?

This PR implements the feature mentioned in #581 

I am open to receive feedbacks and receive suggestions on code improvement

Here's the video demonstration for the same: 

https://www.loom.com/share/b605305cce704c69a7b07b94b5008051

### CODE REVIEW

**Review by a20hek — CHANGES_REQUESTED** [review]
@Sajal0208 i have made some necessary changes to the code

things look good to me, but there is one problem – when I remove member, the list of members does not get updated. i have to refresh the page for it to be updated.

Please fix that, and request a review from me post that

File: src/pages/dashboard/listings/index.tsx [hunk 1]
```diff
@@ -615,21 +615,24 @@ function Bounties() {
...
-                              onClick={() =>
-                                window.open(
-                                  `${router.basePath}/dashboard/listings/${currentBounty.slug}/duplicate`,
-                                  '_blank',
-                                )
-                              }
-                            >
-                              Duplicate
-                            </MenuItem>
+                            {(currentBounty.type === 'bounty' ||
```
Comment (a20hek) [inline src/pages/dashboard/listings/index.tsx:618]: the changes to this file is not from your commit; can you remove this please?

**Review by a20hek — CHANGES_REQUESTED** [review]
rest looks good to me

File: src/pages/dashboard/listings/index.tsx (hunk 1 above)
Comment (Sajal0208) [reply src/pages/dashboard/listings/index.tsx]: Done!

**Review by a20hek — APPROVED** [review]

**Pull request merged**